Gemfile

```diff
@@ -100,7 +100,7 @@ group :development do
 end
 
 group :test do
-  gem 'capybara', '~> 3.7'
+  gem 'capybara', '~> 3.8'
```

Comment:
Thanks @houndci-bot, but I think it's better to stay consistent with the rest of this file, which uses single quotes throughout.

Reply:
I'm going to remove @houndci-bot because it doesn't follow our default RuboCop version and configuration.

Code linters run on Travis, so it is useless